### ISSUE TITLE
Adding an 'empty' user is possible with drone-cli

### ISSUE THREAD

**Issue by frebib** [drone/drone#2738]
```
$ drone user ls
frebib
$ drone user add             
Successfully added user
$ drone user ls

frebib
```

It seems to be possible that when executing the above commands, a user with an empty name is created. It doesn't however seem possible to delete that user in the same manner. In executing this command I was expecting the implicit `--help` to show up, but instead I got this.

Realistically this should be guarded against on the server side, hence why I'm opening this issue here, but I suppose the cli should also prevent the user from doing this. In this situation (and this applies to all drone-cli subcommands) I would expect the cli to show usage for the subcommand and exit `1` when a non-optional argument is not provided such as the `username` in this case.

**Comment by strainovic** [comment]
+1

**Comment by bradrydzewski** [comment]
this was fixed in 1.2.1
